app/styles/themes/gothamist/_theme.legacy-variables.scss

```diff
@@ -98,15 +98,15 @@ $breakpoints: (
 /**
  * Animation
  */
-$hard-ease-in: cubic-bezier(0.860, 0.000, 0.070, 1.000);
+//$hard-ease-in: cubic-bezier(0.860, 0.000, 0.070, 1.000);
 
 
 /**
  * Border Styles
  */
-$border-med: 6px;
-$border-thick: 8px;
-$border-opacity: 1px solid rgba($c-white, .2);
-$border-style: 1px solid $c-border-color;
-$border-style-m: 2px solid $c-border-color;
-$border-style-thick: 3px solid $c-border-color;
+//$border-med: 6px;
+//$border-thick: 8px;
+//$border-opacity: 1px solid rgba($c-white, .2);
+//$border-style: 1px solid $c-border-color;
+//$border-style-m: 2px solid $c-border-color;
+//$border-style-thick: 3px solid $c-border-color;
```

Comment:
If these are safe to delete we should just delete them instead of commenting them out